### PR TITLE
fix(presets): Bug fix for auto-switch theme checkbox values (@amarnathsama)

### PR DESCRIPTION
**Pull request by amarnathsama**
### Description

Fixes bug where applying a preset wouldn't update the auto switch theme light and dark theme checkbox values (pictured below).
image

### Checks

- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

### CODE REVIEW

File: frontend/src/ts/config.ts
Comment (Miodec) [inline frontend/src/ts/config.ts:1]: I would prefer to not add more DOM modifying code in the config module. Please use the config event instead and add this code in settings.ts

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/src/ts/config.ts
Comment (amarnathsama) [reply frontend/src/ts/config.ts:1]: I have made the changes, please let me know if they are good or not.

File: frontend/src/ts/pages/settings.ts [hunk 1]
```diff
@@ -1326,6 +1326,18 @@ ConfigEvent.subscribe((eventKey) => {
   }
 });
 
+ConfigEvent.subscribe((eventKey, eventValue) => {
```
Comment (Miodec) [inline frontend/src/ts/pages/settings.ts:1329]: Please move this inside the subscribe function thats just above.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/src/ts/pages/settings.ts (hunk 1 above)
Comment (amarnathsama) [reply frontend/src/ts/pages/settings.ts]: done

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**